.rubocop.yml

```diff
@@ -1,7 +1,5 @@
 AllCops:
   Exclude:
-    - 'node_modules/**/*'
-    - 'vendor/bundle/**/*'
```

Comment:
I think we should annotate our config a bit more… I was wondering why you removed these lines. Now I quickly found out that these are the defaults (because we've been talking about merging here). In the future though, I think it might not be super-obvious which files/patterns will be ignored.

Maybe add a comment:

```yml
# Add additional exclude patterns for files that should not be analyzed.
# These extend the default RuboCop exclude patterns:
# https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml
AllCops:
  Exclude:
    # …
```